model/ebs_volume.go

```diff
@@ -0,0 +1,51 @@
+package model
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+type EbsVolume struct {
```

Comment:
nit: Can we change `Ebs` to `EBS` so that we can be consistent among it and other symbols like `VPDID`, `RouteTableID`, `NATGateway`, etc?